packages/esm-ward-app/src/hooks/useInpatientRequest.ts

```diff
@@ -6,7 +6,10 @@ import useWardLocation from './useWardLocation';
 const defaultRep =
   'custom:(dispositionLocation,dispositionType,disposition,dispositionEncounter:full,patient:default,dispositionObsGroup,visit)';
 
-export function useInpatientRequest(dispositionType: Array<DispositionType> = ['ADMIT'], rep: string = defaultRep) {
+export function useInpatientRequest(
+  dispositionType: Array<DispositionType> = ['ADMIT', 'TRANSFER'],
```

Comment:
technically I didn't need to change the default value, but if we are going to have a default value seemed to make sense to have it equal to the only value we are currently passing in.